e2e/setup-frontend.sh

```diff
@@ -9,7 +9,7 @@ cd ../apps/extension && \
     export NAMADA_INTERFACE_MASP_PARAMS_PATH=http://localhost:8080/assets/; \
     export NAMADA_INTERFACE_NAMADA_CHAIN_ID=${CHAIN_ID}; \
     export NAMADA_INTERFACE_NAMADA_URL=http://localhost:27657 && \
-    npx webpack --watch &
+    webpack-cli --watch &
```

Comment:
This might run into problems if you're running the version of webpack from the e2e package.json, because the interface uses webpack v5 and the extension uses webpack v4. I think if you run something like:

    NAMADA_INTERFACE_NAMADA_URL=http://localhost:27657 yarn workspace @namada/extension run start:chrome

instead of directly calling webpack it will use the interface/extension webpack, and you wouldn't need to change directory.

Reply:
Cool! I will check it out :)